homeassistant/components/http/__init__.py

```diff
@@ -289,6 +295,19 @@ def register_static_path(self, url_path, path, cache_headers=True):
                               self.ssl_certificate, error)
                 context = None
                 return
+
+            try:
+                if self.ssl_peer_certificate:
+                    context.verify_mode = ssl.CERT_REQUIRED
+                    context.load_verify_locations(
+                        cafile=self.ssl_peer_certificate)
+            except OSError as error:
```

Comment:
This should not happen because we have an `isfile` validator.

Reply:
should I remove it from line 293 too? (I copy-pasted this from above)